### ISSUE TITLE
[Bug] NoSuchMethodError: org.bukkit.entity.LivingEntity.isInvisible() on 1.12.2

### ISSUE THREAD

**Issue by RinNiko** [Insprill/damage-indicator#21]
Paper 1.12.2 Java 8

```
[16:35:56 ERROR]: Could not pass event EntityRegainHealthEvent to DamageIndicator v1.10.3
java.lang.NoSuchMethodError: org.bukkit.entity.LivingEntity.isInvisible()Z
        at com.zenya.damageindicator.event.Listeners.shouldShowHologram(Listeners.java:97) ~[?:?]
        at com.zenya.damageindicator.event.Listeners.onEntityRegainHealthEvent(Listeners.java:79) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor106.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(:70) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at co.aikar.timings.TimedEventExecutor.execute(:78) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.plugin.RegisteredListener.callEvent(:62) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.plugin.SimplePluginManager.callEvent(:513) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.heal(:799) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.heal(:790) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MobEffectList.tick(:60) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MobEffect.b(:111) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MobEffect.tick(:98) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.tickPotionEffects(:575) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.Y(:313) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.Entity.B_(:366) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.B_(:2115) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityHuman.B_(:147) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityPlayer.playerTick(:345) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.PlayerConnection.e(:147) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.NetworkManager.a(:484) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.ServerConnection.c(:182) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(:995) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(:432) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(:778) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(:669) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312] 


[16:32:17 ERROR]: Could not pass event EntityDamageByEntityEvent to DamageIndicator v1.10.3
java.lang.NoSuchMethodError: org.bukkit.entity.LivingEntity.isInvisible()Z
        at com.zenya.damageindicator.event.Listeners.shouldShowHologram(Listeners.java:97) ~[?:?]
        at com.zenya.damageindicator.event.Listeners.onEntityDamageEvent(Listeners.java:50) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1720.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(:70) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at co.aikar.timings.TimedEventExecutor.execute(:78) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.plugin.RegisteredListener.callEvent(:62) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.plugin.SimplePluginManager.callEvent(:513) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.craftbukkit.v1_12_R1.event.CraftEventFactory.callEvent(:95) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.craftbukkit.v1_12_R1.event.CraftEventFactory.callEntityDamageEvent(:622) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.craftbukkit.v1_12_R1.event.CraftEventFactory.handleEntityDamageEvent(:525) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at org.bukkit.craftbukkit.v1_12_R1.event.CraftEventFactory.handleLivingEntityDamageEvent(:654) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.damageEntity0(:1424) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityHuman.damageEntity0(:861) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.damageEntity(:897) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityHuman.damageEntity(:774) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityPlayer.damageEntity(:635) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityMonster.B(:69) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityZombie.B(:202) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.PathfinderGoalMeleeAttack.a(:127) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.PathfinderGoalMeleeAttack.e(:119) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.PathfinderGoalSelector.a(:110) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityInsentient.doTick(:687) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.n(:2341) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityInsentient.n(:509) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityMonster.n(:25) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityZombie.n(:155) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityLiving.B_(:2194) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityInsentient.B_(:248) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.EntityMonster.B_(:29) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.World.entityJoinedWorld(:1855) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.World.h(:1825) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.World.tickEntities(:1632) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.WorldServer.tickEntities(:705) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(:947) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(:432) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(:778) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(:669) ~[patched_1.12.2.jar:git-Paper-e90dfe0]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
```

**Comment by RinNiko** [comment]
How is this going on @Insprill i bought this but can't use it :( I have tried many version.

**Comment by Insprill** [comment]
Fixed in 1.10.4.
